app/Http/Controllers/Wiki/CountriesController.php

```diff
@@ -121,25 +121,31 @@ public function search($search_text = '')
         $limit     = min($limit, 50);
         $page      = checkParam('page') ?? 1;
         $formatted_search = str_replace(' ', '', $search_text);
+
```

Comment:
is there a code change in CountriesController, or just formatting? From what I can tell it is just formatting, but I may be missing something.
If only formatting, please remove from this PR

Reply:
Yes, it is only formatting change. Let me remove it from this PR.